### PR TITLE
Added some more explanations 

### PR DESCRIPTION
**Pull request by aikhan**

### CODE REVIEW

File: Operating Systems/concepts.txt [hunk 1]
```diff
@@ -66,9 +66,9 @@ PARAMETER PASSING: 3 ways to pass parameteres: Registers, Creating a block or ta
 TYPES: Types of system calls are: Process control, maintainance, file management, device management, information maintainance.
 
 LINKER: Linker combines object files by bringing in libraries converting into binary executable file.
-LOADER: Loaders brings the program into memory inorder for it to be executed. Modern systems dont bring in libraries into executable rather use DLLs that are loaded as needed.
+LOADER: Loaders brings the program into memory inorder for it to be executed. Modern systems dont bring in libraries into executable rather use DLLs that are loaded as needed. (Static Libraries are bundled together with the executable and dynamic libraries are linked at runtime, noth have their pros and cons.)
```
Comment (Rubix982) [inline Operating Systems/concepts.txt]: Typo! `noth` -> `both`.

File: Operating Systems/concepts.txt [hunk 2]
```diff
@@ -81,11 +81,11 @@ It is very much needed to seprate these two inorder to reach maximum flexibility
 
 OS STRUCTURE: 5 basic structures: 
 
-MONOLYTHIC: Doesnot break into subsystems and has no distinction between user and kernel mode. Provides direct access to hardware.
+MONOLITHIC: Doesnot break into subsystems and has no distinction between user and kernel mode. Provides direct access to hardware.
 
-LAYERED: Another approach is to create multiple small layers. In which each layer depends upon the services the layer below. No lower layer can use services of upper layer. This can be inefficient as request for service from higher layer has to filter all lower layers.
+LAYERED: Another approach is to create multiple small layers. In which each layer depends upon the services the layer below. No lower layer can use services of upper layer. This can be inefficient as request for service from higher layer has to filter all lower layers but it does provides abstraction makes the code less cohesive.
```
Comment (Rubix982) [inline Operating Systems/concepts.txt]: `but it does provides abstraction makes the code less cohesive.` -> `but it does provide abstraction whihc makes the code less cohesive`.

**Review by Rubix982 — CHANGES_REQUESTED** [review]
Some grammatical/typing fixes suggested.

**Comment by Rubix982** [comment]
Hi Ali! Thanks for the PR! I've left some comments from my review. Feel free to ping when you're done.

**Comment by aikhan** [comment]
Sorry for the typos, I've fixed them now.

Thanks

**Comment by Rubix982** [comment]
LGTM.

@HassanZhd.

**Pull request merged**